README.md

```diff
@@ -22,6 +22,22 @@ To install the package from PIP you can simply do:
 pip install fmeval
 ```
 
+Note: if you you run into the following error while installing `fmeval` on a Mac, please try running the steps below.
```

Comment:
Can you add a troubleshooting header? 
Something like this:

## Troubleshooting